### ISSUE TITLE
Schema errors when importing

### ISSUE THREAD

**Issue by orlp** [moepmoep12/exile-db#2]
It seems there are some schema mismatches:

```
Error while loading table ItemClasses: SqliteError: table ItemClasses has no column named CanUpgradeRarity
Error while loading table ItemCostPerLevel: SqliteError: table ItemCostPerLevel has no column named CostHardmode
Error while loading table LeagueFlag: SqliteError: table LeagueFlag has no column named IsRuthless
Error while loading table MiscAnimated: SqliteError: table MiscAnimated has no column named HASH32
Error while loading table Music: SqliteError: table Music has no column named HASH16
Error while loading table SkillGems: SqliteError: table SkillGems has no column named IsSupport
Error while loading table WorldAreas: SqliteError: table WorldAreas has no column named VaalArea
```

**Comment by moepmoep12** [comment]
The schema changed with the latest patch. I'll release a new version with the latest schema. This will hopefully fix this.

**Comment by moepmoep12** [comment]
Solved by 396231294b49677aa1a2dc3558e3f4cc50ce37d6
